src/routes/events.route.ts

```diff
@@ -0,0 +1,9 @@
+import { Hono } from "hono";
+
+const eventsRoute = new Hono();
+
+eventsRoute.post("/", (c) => {
+  return c.json({ id: "some-uuid", message: "created" }, 201);
+});
+
+export default eventsRoute;
```

Comment:
[nitpick] I would call these routers and not route but that is mainly because of how express called them but if Hono has a different convention then that also works